### ISSUE TITLE
Allow custom ordering of racks in the elevations list view

### ISSUE THREAD

**Issue by chrispcall** [netbox-community/netbox#1755]
Just installed the software and i'm loving it with one visual exception:  It would be nice to have an option to display the rack elevation view mirrored when we are looking at the back view.  I spend most of my time behind my racks and in the current view, I can only look at the the way they are positioned (L-R) in the front view and switching to back view shows me the back of each rack but they are out of order for how I am used to looking at them in real life.

Hope that makes sense.

Basically I would like an option for rack elevation views to look like this:
Front view:
[A-1]   [A-2]   [A-3]   [A-4]

Then back view:
[A-4]   [A-3]   [A-2]   [A-1]

I searched open and closed requests and did not see anything like this.  Also, this is my first github anything, so please go easy on my ignorance!

### Issue type
[X ] Feature request 
[ ] Bug report      
[ ] Documentation   

### Environment
* Python version:  3.5.2
* NetBox version:  2.2.6

**Comment by jasonguy** [comment]
I too would like a view to see multiple racks in a view. Generally a DC is organized in rows of racks. Perhaps a designation between Site and Rack needs to be created, called "Row"? Then you can have a "Row View", which would show fronts and backs as mentioned. Another way to do it is have a button to display the fronts, or the backs, which limits how busy the screen will get.

**Comment by chrispcall** [comment]
@jasonguy Are you talking about something different than elevation view?  That shows you the racks in a row.  I'm just looking for a small feature WITHIN that view.

**Comment by jasonguy** [comment]
@chrispcall My mistake. I had not created multiple racks yet. I agree that the "rear" view should be reversed, as if you "walked around the back", everything would be reversed left to right. Good idea...I deleted my useless comment. :)

**Comment by jeremystretch** [comment]
Rather than simply flipping the ordering, we probably want to expose a way for the user to choose how the racks are sorted. (This is accomplished in most list views using django-tables' built-in `RequestConfig()`, which allows sorting by column header.)

**Comment by DerDanilo** [comment]
Since this feature request was accepted we're wondering when it's going to be implemented. 
Any news on this? This doesn't sound to difficult to do but I may be wrong.

**Comment by jas0nat0r** [comment]
We'd like for this feature to accomplish 2 things.

1.  Allow you to specify the sort order per rack group.  For example, if we number racks based on their relative position to a defined wall then opposing rows would have 1-4 going left to right and opposite row would have 1-4 going right to left
 
back       front       back
          [1]           [1]
          [2]           [2]
          [3]           [3]
          [4]           [4]

2.  Reverse specified sort order when selecting "rear" option.  This would allow you to retain the physical layout of the rack and you look at it from the back.
